dev/vcpkg/env.sh

```diff
@@ -10,7 +10,7 @@ SCRIPT_ROOT="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
 
 export VCPKG_ROOT="$SCRIPT_ROOT/.vcpkg"
 export VCPKG="$SCRIPT_ROOT/.vcpkg/vcpkg"
-export VCPKG_TRIPLET=x64-linux-avx
+export VCPKG_TRIPLET=$([ "${CPU_TARGET:-}" = "aarch64" ] && echo "arm64-linux-release" || echo "x64-linux-avx")
```

Comment:
is the triplet `aarch64` exist?

Reply:
Ignore it, I find it here
https://github.com/microsoft/vcpkg/blob/master/triplets/community/arm64-linux-release.cmake